atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -283,6 +285,11 @@ private void checkLockAndTimestampBlocks() {
     }
 
     private void checkNamespaceConfig() {
+        getNamespaceString();
+    }
+
+    @Value.Derived
+    public String getNamespaceString() {
```

Comment:
Is any of the config in here live reloadable?  Just wondering if there's a way we can only calculate this once rather doing this all the time.  Don't feel strongly about it though

Reply:
Yes, this is not live-reloadable, feels like a good idea to not repeat the check multiple times.